static/js/pages/independent-expenditures.js

```diff
@@ -66,6 +66,7 @@ $(document).ready(function() {
   new tables.DataTable($table, {
     title: 'Independent expenditure',
     path: 'schedules/schedule_e',
+    query: {is_notice: 'false'},
```

Comment:
I thought you wanted to make the change on the frontend. And yes, this needs to be added to all tables that hit `/schedules/schedule_e/`, unless you want to hide notices in some views but not others.